airbyte-integrations/bases/source-acceptance-test/README.md

```diff
@@ -48,7 +48,7 @@ _Note: this will assume that docker image for connector is already built_
 
 Using Gradle
 ```bash
-./gradlew :airbyte-integrations:connectors:source-<name>:standardTest
+./gradlew :airbyte-integrations:connectors:source-<name>:sourceAcceptanceTest
```

Comment:
I believe this is the right command.

Reply:
@keu does this make sense? the previous command did not exist and I was poking through code for the right one, so might be wrong.

Reply:
yes, you are right 👍